onnxruntime/python/tools/kernel_explorer/kernels/vector_add_test.py

```diff
@@ -69,16 +72,29 @@ def profile_vector_add_func(size, dtype, func):
     print(dtype, size, f, f"{t*1000:.2f} us", f"{size*3*(dtype_to_bytes(dtype))*1e3/t/1e9:.2f} GB/s")
 
 
+def profile_with_args(size, dtype):
+    for func in dtype_to_funcs(dtype):
+        profile_vector_add_func(size, dtype, func)
+    print()
+
+
 def profile():
     sizes = [10000, 100000, 1000000, 10000000]
-    dtypes = ["float16", "float32"]
     for dt in dtypes:
         for s in sizes:
-            for f in dtype_to_funcs(dt):
-                profile_vector_add_func(s, dt, f)
-            print()
+            profile_with_args(s, dt)
         print()
 
 
 if __name__ == "__main__":
-    profile()
+    import argparse
+
+    parser = argparse.ArgumentParser()
+    group = parser.add_argument_group("profile with args")
+    group.add_argument("size", type=int)
+    group.add_argument("dtype", choices=dtypes)
+    if len(sys.argv) == 1:
+        profile()
+    else:
+        args = parser.parse_args()
```

Comment:
add an example command as comments (and for other files as well)?

Reply:
@zhangyaobit It is not needed, `ArgumentParser` will handle it.

Reply:
how about adding some example commands in the readme?

We can follow some of the style of this page: https://github.com/microsoft/onnxruntime/tree/main/onnxruntime/python/tools/transformers

Reply:
I think adding help doc strings to argparser is sufficient. But they are not necessary. The semantics for them is pretty simple and it is easy to play around.